sql-plugin/src/main/scala/com/nvidia/spark/rapids/window/GpuWindowExpression.scala

```diff
@@ -1711,6 +1711,7 @@ class RankFixer extends BatchedRunningWindowFixer with Logging {
   override def close(): Unit = {
     previousRank.foreach(_.close())
```

Comment:
Should we also change?

```suggestion
    previousRank.foreach(_.safeClose())
```

Reply:
Ah, it turns out that `cudf.Scalar` objects cannot be `safeClose()`d:
```
Error: ] /home/runner/work/spark-rapids/spark-rapids/sql-plugin/src/main/scala/com/nvidia/spark/rapids/window/GpuWindowExpression.scala:1712: value safeClose is not a member of ai.rapids.cudf.Scalar
2345
Error: [ERROR] one error found
```